src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -154,6 +154,8 @@ def load_arguments(self, _):
         c.argument('dns_name_prefix', options_list=['--dns-name-prefix', '-p'])
         c.argument('generate_ssh_keys', action='store_true', validator=validate_create_parameters)
         c.argument('node_vm_size', options_list=['--node-vm-size', '-s'], completer=get_vm_size_completion_list)
+        c.argument('nodepool_name', type=str, default='nodepool1',
+                   help='node pool name, truncated to 12 characters')
```

Comment:
Would a validator that checks that the nodepool_name is at most 12 characters be helpful here?

Reply:
Maybe start the sentence with a uppercase / capital letter?

Reply:
Maybe start the sentence with a uppercase / capital letter?